mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultInfo.tsx

```diff
@@ -0,0 +1,62 @@
+import { View } from '@components'
+import { ThemedProps, ThemedText } from '@components/themed'
+import { TokenIconGroup } from '@components/TokenIconGroup'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import NumberFormat from 'react-number-format'
+
+interface VaultInfoProps {
+  label: string
+  value?: BigNumber
+  tokens?: string[]
+  valueType: VaultInfoValueType
+  prefix?: string
+  suffix?: string
+  decimalPlace?: number
+  valueThemedProps?: ThemedProps
+}
+
+type VaultInfoValueType = 'NUMBER' | 'TOKEN_ICON_GROUP'
```

Comment:
can we use enum here?

Reply:
I have no preference, but curious to know is there an advantage of using `enum` here? My main purpose is just differentiating possible input types, I don't plan to iterate it

Reply:
My suggestion here to use enum is to avoid writing the value at multiple places

Reply:
Isn't the auto code completion available for both options? You don't really need to manually type the value for both cases. But for enum you'll need to always import/export it 
![image](https://user-images.githubusercontent.com/40191153/139998132-c47399a5-1763-4dae-8114-bbaedc5d2ff1.png)